apps/climatemappedafrica/package.json

```diff
@@ -34,6 +34,7 @@
   },
   "dependencies": {
     "@apollo/client": "catalog:",
+    "@badrap/bar-of-progress": "catalog:",
```

Comment:
Nice idea keeping the progress indicator at the top, just wondering if the [Linear Progress indicator](https://mui.com/material-ui/react-progress/?srsltid=AfmBOooQPLjRHLAtTxbq6uZfVQTSvixPN-dbS6SxEIHfC3i3_QVI_pFu#linear) can't do the same this library does ?